specs/chrome_extension_content_scripts.table

```diff
@@ -0,0 +1,17 @@
+table_name("chrome_extension_content_scripts")
+description("Chrome browser extension content scripts.")
+schema([
+    Column("identifier", TEXT, "Extension identifier"),
+    Column("version", TEXT, "Extension-supplied version"),
+    Column("script", TEXT, "The content script used by the extension"),
+    Column("match", TEXT, "The pattern that the script is matched against"),
+])
+attributes(user_data=True)
```

Comment:
If you need UID for this table to work then those columns must exist in the spec and have the attribute `index=True`.